tests/test_dashboard.py

```diff
@@ -8,11 +8,22 @@
     pip install pytest-playwright
```

Comment:
I'm not an expert on `streamlit`. Maybe, could ask @stefsmeets  for a quick look at your suggested changes.

Reply:
Happy to have a look, let me know which bits.

Reply:
Thanks @elboyran for the comment. the dashboard is created using streamlit, see [here](https://github.com/dianna-ai/dianna/tree/main/dianna/dashboard). 
@stefsmeets can you please have a look at the updated docstring [here](https://github.com/dianna-ai/dianna/pull/728/files/10a9ba169446c1813dd60ef601b9107360982ec1#diff-37f37eddbedf6e2deac58b85ba731696800ae8670a251d57d6cc5bde95e8baa1L1-L16)? Together with @cwmeijer, I found that the old docstring is not complete, the  `playwright codegen http://localhost:8501` returns error because first the server should be started.  So I updated the docstring saying to use `streamlit` instead of `playwright` to run tests.